index.html

```diff
@@ -122,6 +122,32 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *Your_Name* card START ________  -->
+        <div class="card">
+          <p class="name">Le Chi Bao</p>
+          <p class="contact">
+            
+            <p >Your handle</p>
```

Comment:
Since these lines make no sense & does not rely on the template, kindly remove these lines.